src/Evolution/EventsAndDenseTriggers/Tags.hpp

```diff
@@ -31,8 +31,43 @@ struct EventsAndDenseTriggers : db::SimpleTag {
   using type = evolution::EventsAndDenseTriggers;
   using option_tags = tmpl::list<OptionTags::EventsAndDenseTriggers>;
   static constexpr bool pass_metavariables = false;
-  static type create_from_options(const type& events_and_triggers) {
-    return deserialize<type>(serialize(events_and_triggers).data());
+  static type create_from_options(
+      const std::vector<evolution::DenseTriggerAndEventsConstruction>&
+          construction) {
+    // The deserialize(serialize()) is to deal with unique_ptrs in the data
+    // structures since we don't have `get_clone` implemented for
+    // triggers/events
+    return deserialize<type>(
+        serialize(
+            type{deserialize<
```

Comment:
I don't think you need to go through the serialization framework twice.  You already have an rvalue after the inner deserialize.